test/common/config/config_provider_impl_test.cc

```diff
@@ -632,6 +628,7 @@ class DeltaDummyConfigProviderManager : public ConfigProviderManagerImplBase {
                           const std::string&,
                           const Envoy::Config::ConfigProviderManager::OptionalArg&) override {
     DeltaDummyConfigSubscriptionSharedPtr subscription =
+
```

Comment:
probably remove this newline? i don't think we tend to have newlines in the middle of declarations

Reply:
done